optaplanner-examples/data/tsp/unsolved/americanRoadTrip-road-km-n50.json

```diff
@@ -0,0 +1,3310 @@
+{
+  "id": 0,
+  "name": "americanRoadTrip-road-km-n50",
+  "distanceType": "ROAD_DISTANCE",
+  "distanceUnitOfMeasurement": "meter",
+  "locationList": [
+    {
+      "@type": "road",
+      "@id": "0",
+      "id": 0,
+      "name": "USS Alabama, Battleship Parkway, Mobile, AL",
+      "latitude": 30.681803,
+      "longitude": -88.014426,
+      "travelDistanceMap": {
+        "1": 2646623.0,
```

Comment:
I still don't like this. This is the `id`, not `@id`, therefore there is inconsistency in referencing here.